src/coord/src/coord.rs

```diff
@@ -3251,9 +3281,10 @@ impl Coordinator {
     fn determine_timestamp(
         &mut self,
         session: &Session,
-        uses_ids: &[GlobalId],
+        unmaterialized_source_ids: &[GlobalId],
+        index_ids: &[GlobalId],
```

Comment:
Not high priority, but should these just be `storage_ids` and `compute_ids`, or is are the names good? Just thinking .. idk if the source ids are unmaterialized or not, just that we are asking for them to be included in the timestamp reasoning.

Reply:
Yeah, they should definitely pivot to something more consistent. Possibly even the `IdBundle` thing from above! I just used the existing local variable names out of laziness.